lighthouse-core/gather/gatherers/image-elements.js

```diff
@@ -162,6 +185,53 @@ function determineNaturalSize(url) {
   });
 }
 
+/**
+ * @param {LH.Crdp.CSS.CSSStyle} [style]
+ * @param {string} property
+ * @return {string | undefined}
+ */
+function findSizeDeclaration(style, property) {
+  if (!style) return;
+  const definedProp = style.cssProperties.find(({name}) => name === property);
+  if (!definedProp) return;
+  return definedProp.value;
+}
+
+/**
+ * Finds the most specific directly matched CSS font-size rule from the list.
+ *
+ * @param {Array<LH.Crdp.CSS.RuleMatch>} [matchedCSSRules]
+ * @param {string} property
+ * @returns {string | undefined}
+ */
+function findMostSpecificCSSRule(matchedCSSRules, property) {
+  /** @type {function(LH.Crdp.CSS.CSSStyle): string | undefined} */
```

Comment:
is this type comment necessary?

Reply:
should be able to get away with 
```suggestion
  /** @param {LH.Crdp.CSS.CSSStyle} declaration */
```

Reply:
if I don't put that type comment, I get a ts warning/error that says `Parameter 'declaration' implicitly has an 'any' type`, should I use a `ts-ignore`?

Reply:
@lemcardenas that happens even with my suggestion?

Reply:
@patrickhulce ah my bad I didn't refresh the page to see your suggestion! it works now!